readthedocs/builds/tasks.py

```diff
@@ -163,3 +169,40 @@ def archive_builds_task(days=14, limit=200, include_cold=False, delete=False):
                     build.commands.all().delete()
             except IOError:
                 log.exception('Cold Storage save failure')
+
+
+def delete_inactive_external_versions(limit=200, days=30 * 3):
```

Comment:
Not sure if we really need a limit here -- are we worried about something specific with it?

Reply:
After 90 days, we may have a lot of versions to delete (or maybe not? We don't know how many PRs are open in a day), the task may be running for a long period of time, but not sure if that's a problem, let me know if you want me to remove it.

Reply:
I think its fine to have a limit, I guess. We can adjust if we need to later.